.github/workflows/build_and_test.yml

```diff
@@ -347,8 +347,8 @@ jobs:
 
     - name: Publish assets
       env:
-        AWS_ACCESS_KEY_ID: ${{ secrets.aws_user_circleci_rocketchat_access_key_id }}
-        AWS_SECRET_ACCESS_KEY: ${{ secrets.aws_user_circleci_rocketchat_secret_access_key }}
+        AWS_ACCESS_KEY_ID: ${{ secrets.aws_user_github_rocketchat_access_key_id }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.aws_user_github_rocketchat_secret_access_key }}
```

Comment:
In "Rotate  AWS key " job you use upper case, here lower case? I don't know if this is case sensitive or not, but please use the real case of the secret key.

Reply:
It is not case sensitive in meaning that it makes it upper case  when you work with  secrets. Already tested it with samw

Reply:
it is made as it was made before - just changed the name  of the variable.